src/ContentAPI.ts

```diff
@@ -20,6 +19,7 @@ import {
 import { Readable } from 'stream'
 import { DeploymentData } from './utils/DeploymentBuilder'
 import { DeploymentOptions } from './ContentClient'
+import { RequestOptions } from 'dcl-catalyst-commons/dist/utils/FetcherConfiguration'
```

Comment:
This is not ok. We probably didn't export `FetcherConfiguration` correctly on `dcl-catalyst-commons` (same on the ones below)